.github/workflows/windows-qe-tpl.yml

```diff
@@ -107,7 +107,7 @@ jobs:
           -e ARM_CLIENT_SECRET='${{secrets.ARM_CLIENT_SECRET}}' \
           -e AZURE_STORAGE_ACCOUNT='${{ secrets.AZURE_STORAGE_ACCOUNT }}' \
           -e AZURE_STORAGE_KEY='${{ secrets.AZURE_STORAGE_KEY }}' \
-          quay.io/redhat-developer/mapt:v0.6.9 azure \
+          quay.io/redhat-developer/mapt:v0.7.4 azure \
```

Comment:
With the newer version we need to adjust the params involved:

https://github.com/redhat-developer/mapt/blob/main/docs/azure/windows.md

At least now we need also the `--nested-virt` + I would suggest to add also `--memory`  and `--cpus` as with those params the search for spot instances is not limited to the default vmsize but to any size matching the requirements